jormungandr/src/jrpc/mod.rs

```diff
@@ -43,6 +43,8 @@ pub enum Error {
     Fragment(FragmentsProcessingSummary),
     #[error("Cound not decode Ethereum transaction bytes, erorr: {0}")]
     TransactionDecodedErorr(String),
+    #[error("We are not supporting mining functionality")]
+    MiningAreNotAllowed,
```

Comment:
why not _is_ instead of _are_? 

I also think something like:

> "Mining is not currently supported" 

Would be a more standard error message, but it's mostly nitpicking.